web3.md

```diff
@@ -823,12 +830,35 @@ loadCallSettings
  // ---------------------------------
     rule <k> #prepareTx TXID:Int
           => #clearLogs
-          ~> #loadCallSettings TXID
-          ~> #executeTx TXID
-          ~> #makeTxReceipt TXID
+          ~> #validateTx TXID
          ...
          </k>
 
+    syntax KItem ::= "#validateTx" Int
+ // ----------------------------------
+    rule <k> #validateTx TXID => . ... </k>
+         <statusCode> ( _ => EVMC_OUT_OF_GAS) </statusCode>
+         <schedule> SCHED </schedule>
+         <message>
+           <msgID>      TXID     </msgID>
+           <txGasLimit> GLIMIT   </txGasLimit>
+           <data>       DATA     </data>
+           ...
+         </message>
+      requires ( GLIMIT -Int G0(SCHED, DATA, false) ) <Int 0
```

Comment:
Should this actually say `G0(SCHED, DATA, ACCTTO ==K .Account)`?

Reply:
Yes it should, for both rules.